Beta_Integrations/GmailSingleUser/GmailSingleUser.yml

```diff
@@ -0,0 +1,211 @@
+category: Messaging
```

Comment:
should this be "Email Gateway"?
@Arsenikr WDYT?

Reply:
Message is ok. This is also where we have mail listener/sender.